base/float.jl

```diff
@@ -836,12 +836,12 @@ number of significand digits in that base.
 """
 function precision end
 
-_precision(::Type{Float16}) = 11
-_precision(::Type{Float32}) = 24
-_precision(::Type{Float64}) = 53
+precision_with_base_2(::Type{Float16}) = 11
```

Comment:
I would keep a leading underscore here

Reply:
Are underscore prefixes really still necessary now that we have `public`?